web/src/routes/admin/user-management/+page.svelte

```diff
@@ -191,6 +192,15 @@
                   >{immichUser.email}</td
                 >
                 <td class="hidden sm:block w-3/12 text-ellipsis break-all px-2 text-sm">{immichUser.name}</td>
+                <td class="hidden xl:block w-3/12 2xl:w-2/12 text-ellipsis break-all px-2 text-sm">
+                  <div class="container mx-auto flex flex-wrap justify-center">
+                    {#if immichUser.quotaSizeInBytes && immichUser.quotaSizeInBytes > 0}
+                      <Icon path={mdiCheck} size="16" />
```

Comment:
Would it not be better to have the actual quota value in the table itself, especially since it is not very wide.